### DISCUSSION TITLE
Links to resolved PR discussions don't open the resolved conversation

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#139515] [Mobile] [answered]
GitHub Mobile beta for Android 

1. Open this link (e.g. from a GitHub notification email):
https://github.com/traefik/traefik/pull/11053#discussion_r1771722992

# Expected results
![Screenshot_20240923-123019.png](https://github.com/user-attachments/assets/5a140aa0-54e7-495c-ac21-36ec064f90d4)

# Actual results

A view showing lots of collapsed discussions and you have to open each one manually and try to guess if it was the right one

![Screenshot_20240923-123151.png](https://github.com/user-attachments/assets/025fdc89-b428-4477-a35f-e3fa764bb8c0)

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback @jsoref , I'll open an internal issue for our team.
